lms/djangoapps/shoppingcart/models.py

```diff
@@ -1928,7 +1928,9 @@ def purchased_callback(self):
 
     def additional_instruction_text(self):
         verification_reminder = ""
+        refund_reminder_msg = _("You have up to two weeks into the course to unenroll and receive a full refund. ")
```

Comment:
I'm not sure where this text came from, but I'd change it a little:

You can unenroll in the course and receive a full refund for 14 days after the course start date.